deployment/legacy-migration/maintenance-worker/migration/mongo/.env

```diff
@@ -12,6 +12,6 @@
 # On Windows this resolves to C:\Users\dornhoeschen\AppData\Local\Temp\mongo-export (MinGW)
 EXPORT_PATH='/tmp/mongo-export/'
 EXPORT_MONGOEXPORT_BIN='mongoexport'
-MONGO_EXPORT_SPLIT_SIZE=100
+MONGO_EXPORT_SPLIT_SIZE=4000
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-08-28T17:06:16Z" title="Wednesday, August 28th 2019, 7:06:16 pm +02:00">Aug 28, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -12,6 +12,6 @@
 # On Windows this resolves to C:\Users\dornhoeschen\AppData\Local\Temp\mongo-export (MinGW)
 EXPORT_PATH='/tmp/mongo-export/'
 EXPORT_MONGOEXPORT_BIN='mongoexport'
-MONGO_EXPORT_SPLIT_SIZE=100
+MONGO_EXPORT_SPLIT_SIZE=4000
```


code creep - but actually I think that's a better default